### PR TITLE
Address management  and  payment integration

### PR DESCRIPTION
**Pull request by Sharmaryan**
- added address management with crud operation
- added payment integration
- added thank you page

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-REACT_APP_JWT_SECRET=qtnxfigawpyrokmejuhc
\ No newline at end of file
+REACT_APP_JWT_SECRET=qtnxfigawpyrokmejuhc
```
Comment (OmKakatkar) [inline .env:1]: Avoid committing .env files

**Review by ashishkatiyar2404 — APPROVED** [review]
UI looks good. The functionality of your code is understandable.
you can merge now.

and work on other functionality of project like - price details etc..

**Review by preethasri — APPROVED** [review]
Good to merge 👍

**Comment by SJTGSHIVAM** [comment]
Looks good

**Review by SJTGSHIVAM — APPROVED** [review]

**Review by OmKakatkar — APPROVED** [review]
Like the use of index files for imports.
Code looks clean
Do check the suggestion below

**Review by samyakshah3008 — APPROVED** [review]
Good for approval

**Comment by Himanshu-Dhawale** [comment]
Good to merge

**Review by Kavita539 — APPROVED** [review]
code looks good, good to merge

**Review by Himanshu-Dhawale — APPROVED** [review]

**Pull request merged**